### DISCUSSION TITLE
Any chance to use a collection instead of a model repository?

### DISCUSSION THREAD

**Discussion by faridsa** [rappasoft/laravel-livewire-tables#200] [Q&A]
Need to render a collection from different custom joins between several models, and I see that this component should do a nice view. But as it works only with a model and relationships I can't manage to fit my data into it.

**Comment by rappasoft** [comment]
I will add collection or query support to the roadmap, shouldn't be too hard with how we have it now.

**Comment by bdelamatre** [comment]
Would pagination, sort and search all be in scope?

**Comment by lironsam** [comment]
Any news from this ? :)

**Comment by tuluong-vl** [comment]
I also need collection support to work with external  Restful Api data result
when will we have this feature?
Thanks

**Comment by tuluong-vl** [comment]
@rappasoft do you have roadmap for this feature?

**Reply by rappasoft** [reply]
This feature is bigger than just changing a few lines. I haven't had time to think about it. If anyone wants to dive in feel free to make a PR.
